src/main/resources/application.properties

```diff
@@ -1,6 +1,6 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/kadas?serverTimezone=UTC
 spring.datasource.username=root
-spring.datasource.password=1234
+spring.datasource.password=${DB_PASSWORD}
```

Comment:
by default spring cannot resolve such placeholders here from your .env file. Did you check it's working?

Reply:
Checked it in different ways. Maybe there's something i misunderstand. Would appreciate more information of this issue.